classes/article/ArticleHTMLGalley.inc.php

```diff
@@ -5,6 +5,7 @@
  *
  * Copyright (c) 2013-2014 Simon Fraser University Library
  * Copyright (c) 2003-2014 John Willinsky
+ * Copyright (c) 2014 Eirik Hanssen
```

Comment:
Sure, I'll just remove it from the copyright section. My name is in the description before the function anyway.